src/prisma/schema.prisma

```diff
@@ -14,12 +14,14 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+
 model User {
   id           Int      @id @default(autoincrement())
   email        String   @unique
   passwordHash String
   isActive     Boolean  @default(true)
   Profile      Profile?
+  Posts        Post[]   @relation("UserToPost")
```

Comment:
Posts should be an optional field, so that we don't get errors when a user has no posts.